physics/module_bl_mynn.F90

```diff
@@ -523,9 +456,9 @@ SUBROUTINE mynn_bl_driver(            &
     REAL,    INTENT(in) :: closure
 
     LOGICAL, INTENT(in) :: FLAG_QI,FLAG_QNI,FLAG_QC,FLAG_QNC,&
-                           FLAG_QNWFA,FLAG_QNIFA
+                           FLAG_QNWFA,FLAG_QNIFA,FLAG_OZONE
 
-    LOGICAL, INTENT(IN) :: mix_chem,fire_turb
+    LOGICAL, INTENT(IN) :: mix_chem,fire_turb,rrfs_smoke
```

Comment:
Here and below, I don't think we need rrfs_smoke because this smoke functionality simply leverages the chem mixing routine, so  if rrfs_smoke is set to true in the namelist, then mix_chem should be set to true somewhere upstream of the MYNN. That is, the MYNN doesn't need to even know about this parameter.

Reply:
These changes are necessary to run on Cheyenne with the gnu compiler. If you want to try reducing the number of changes without causing the model to crash, they you're welcome to do so... if you have a Cheyenne account.